Dockerfile

```diff
@@ -1,11 +1,14 @@
-#syntax=docker/dockerfile:1.4.2
+#syntax=docker/dockerfile:1.5.1
 
-FROM golang:1.19-alpine AS build
-WORKDIR /go/src/
+FROM cgr.dev/chainguard/go:latest AS build
```

Comment:
I'm not aware of chainguard, and would rather stick to the bog standard docker hub images (or a trustworthy mirror there of)